### ISSUE TITLE
Having issue adding a data disk to an existing scale set(created by ACS)

### ISSUE THREAD

**Issue by senpx** [Azure/vm-scale-sets#37]
Having issue adding a data disk to an existing scale set. and this vmss created by ACS swarm cluster. there is no option in GUI to add an additional disk to each instance of VM. Below I got it from CLI. Any recommendations or inputs.?

"az vmss disk attach -g dcospx -n dcospx --size-gb 100 --lun 3
The Resource 'Microsoft.Compute/virtualMachineScaleSets/dcospx' under resource group 'dcospx' was not found."

**Comment by gbowerman** [comment]
Hi, a couple of things to check..
1. Double check in the portal that there is a scale set called dcospx in resource group dcospx.
2. You can only add data disks to scale sets that were created based on Azure Managed Disks. If your ACS cluster created a scale set based on unmanaged disks (i.e. you see a set of 5 storage accounts that were created for the scale set) then you can't add attached disks for it.

**Comment by senpx** [comment]
Thanks for your response. So by default ACS cluster creates unmanaged disk and not able to add the more disk to it. 

Is it a limitation? Then acs cluster use case will less. 

 

Thanks again 

 

Sen S

**Comment by gbowerman** [comment]
@senpx No. My response was that by default ACS **does** create Managed Disk based scale sets for DOCS and Swarm.
